understanding/media-alternative-prerecorded.html

```diff
@@ -58,114 +56,80 @@
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.8:</span>Media Alternative (Prerecorded) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.8:</span>メディアに対する代替 (収録済) (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>An <a href="#dfn-alternative-for-time-based-media">alternative for time-based media</a> is provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-synchronized-media">synchronized media</a> and for all prerecorded <a href="#dfn-video-only">video-only</a> media.
-                     
+                  <p>すべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-synchronized-media">同期したメディア</a>及びすべての収録済の<a href="#dfn-video-only">映像しか含まない</a>メディアに対して、<a href="#dfn-alternative-for-time-based-media">時間依存メディアに対する代替</a>コンテンツが提供されている。
+
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Precorded videos can be understood by more people.</dd>
+                  <dt>目標</dt>
+                  <dd>収録済の映像を、より多くの人が理解できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide a text equivalent for all content in videos.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>映像のすべての内容と同等のテキストを提供する。</dd>
```

Comment:
「映像のすべての内容」は「映像の一部ではなく全部」というニュアンスに読めて、すこし微妙なように思いました。「映像のすべてのコンテンツ」とするか、「映像に含まれるすべての内容」とした方が良いかもしれません。
また、text equivalent は text alternative(テキストによる代替)の言い換えなので、「テキストによるXX」と訳せるとよさそうなのですが、なかなか難しいですね……。ちょっと補っても良いならこうでしょうか。

```suggestion
                  <dd>映像に含まれるすべての内容に対して、テキストによる同等の内容を提供する。</dd>
```